src/ec/suite_b/ops/ops.rs

```diff
@@ -215,11 +210,9 @@ impl CommonOps {
 
     pub fn elem_verify_is_not_zero(&self, a: &ElemUnreduced)
                                    -> Result<(), error::Unspecified> {
-        match unsafe {
-            GFp_constant_time_limbs_are_zero(a.limbs.as_ptr(), self.num_limbs)
-        } {
-            0 => Ok(()),
-            _ => Err(error::Unspecified),
+        match limbs_are_zero_constant_time(&a.limbs[..self.num_limbs]) {
+            false => Ok(()),
+            true => Err(error::Unspecified),
```

Comment:
No worries. I'll try to see if I can simplify it myself. I think there's other code that can benefit from such a `BooleanMask` type.